privacyidea/lib/eventhandler/federationhandler.py

```diff
@@ -56,6 +56,8 @@ class FederationEventHandler(BaseEventHandler):
     description = "This event handler can forward the request to other " \
                   "privacyIDEA servers"
 
+    # TODO: Do we need to change the federation handler this way, that it does only pre-handling?
```

Comment:
Ah, one remaining thing: I guess we shouldn't commit TODOs. Is it even possible to turn the federation handler, as it is designed currently, into a pre-handler? Since the pre-event handler cannot modify the response.
I guess we could design a slightly different federation pre-handler which contacts the remote server if there is no matching local token and automatically enrolls a remote token, which could then be subsequently found by the /validate/check?

Reply:
So what do you want me to do?
Remove the todo?

We had a lot of ``#TODO``s in the past.

Reply:
Hm -- maybe remove the TODO and open an issue? We could then also discuss the design of such a pre-event handler there.

Reply:
Can we meet in the middle, leave the #TODO and open an issue?

Reply:
OK, sounds good!

Reply:
I added #1164

Reply:
:+1: